seed/tests/test_labels.py

```diff
@@ -106,10 +106,10 @@ def test_error_occurs_when_trying_to_apply_a_label_to_taxlotview_from_a_differen
         with transaction.atomic():
             with self.assertRaises(IntegrityError):
                 self.api_view.add_labels(
-                    self.api_view.models['taxlot'].objects.none(),
-                    'taxlot',
-                    [org_1_taxlot.id],
-                    [self.org_2_status_label.id]
+                    qs=self.api_view.models['taxlot'].objects.none(),
+                    inventory_type='taxlot',
+                    inventory_ids=[org_1_taxlotview.id],
```

Comment:
The one important change is this line, we were passing canonical IDs when it needed to be view IDs.

Test was failing for me b/c I was using `--keepdb` flag when running test. Previously this passed b/c the IDs of views and canonical inventory were "synched". Same goes for the other label test I modified